fixtures/benchmarks/src/benchmarks.udl

```diff
@@ -0,0 +1,40 @@
+namespace benchmarks {
+    // Run all benchmarks and print the results to stdout
+    void run_benchmarks(string language_name, TestCallbackInterface cb);
```

Comment:
nit: very odd indentation all over this file (which I think is true in most of our udl files, but the comments in particular leapt out at me)